packages/backend-defaults/config.d.ts

```diff
@@ -740,8 +740,53 @@ export interface Config {
           connection: string;
           /** An optional default TTL (in milliseconds). */
           defaultTtl?: number | HumanDuration | string;
+        }
+      | {
+          store: 'infinispan';
+          /**
+           * Configuration for the Infinispan cache store.
+           */
+          infinispan?: {
```

Comment:
Is there no canonical single connection string that people can use for convenience?